manifest.json

```diff
@@ -3,8 +3,11 @@
   "name": "Active Yuwa",
   "version": "0.1.0",
   "description": "Ask Buddie extension to improve the post reach.",
+  "permissions": ["scripting", "activeTab", "storage"],
+"host_permissions": [
```

Comment:
Can you fix the formatting please, I will add the formatting & lint config soon.